myeongseok/app.js

```diff
@@ -35,23 +37,58 @@ app.get('/ping', (req, res, next) => {
   res.status(200).json({ message: 'pong' });
 });
 
-app.post('/users/signup', (req, res, next) => {
+app.post('/users/signup', async (req, res, next) => {
   const { name, email, profileImage, password } = req.body;
+  const saltRounds = 12;
 
-  myDataSource.query(
+  const hash = await bcrypt.hash(password, saltRounds);
```

Comment:
```suggestion
  const hashedPassword = await bcrypt.hash(password, saltRounds);
```
좀 더 정확하게 네이밍할 필요가 있어보입니다.

Reply:
네이밍 변경했습니다!